pkg/trafficmngr/nftables/utils.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2024 flannel authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build !windows
+// +build !windows
+
+package nftables
+
+import (
+	"context"
+
+	log "k8s.io/klog/v2"
+	"sigs.k8s.io/knftables"
+)
+
+const (
+	masqueradeTestTable = "masqueradeTest"
+)
+
+func (nftm *NFTablesManager) checkRandomfully(ctx context.Context) bool {
```

Comment:
If you could add one-liner describing this function, that'd be great

Reply:
done